core/pkg/store/query.go

```diff
@@ -25,7 +25,7 @@ const flagSetIdKeySourceCompoundIndex = flagSetIdIndex + "+" + keyIndex + "+" +
 
 // flagSetId defaults to a UUID generated at startup to make our queries consistent
 // any flag without a "flagSetId" is assigned this one; it's never exposed externally
-var nilFlagSetId = uuid.New().String()
+var NilFlagSetId = uuid.New().String()
```

Comment:
i hate this change, but i need this nilflagsetid in the evaluator tests somewhere - maybe this is a sign, that our implementation does not follow separation as good as we would love it to be.